src/wallet/rpcwallet.cpp

```diff
@@ -430,6 +439,8 @@ static UniValue sendtoaddress(const JSONRPCRequest& request)
             "       \"" + FeeModes("\"\n\"") + "\""},
                     {"avoid_reuse", RPCArg::Type::BOOL, /* default */ "true", "(only available if avoid_reuse wallet flag is set) Avoid spending from dirty addresses; addresses are considered\n"
             "                             dirty if they have previously been used in a transaction."},
+                    {"verbose", RPCArg::Type::BOOL, /* default */ "false",
+                            "Whether to display the fee reason or not."},
```

Comment:
It  would be preferable to have this text be generic instead of specific for the fee reason. Same for the other RPCs you've changed.